.github/workflows/continous-integration.yml

```diff
@@ -145,10 +145,11 @@ jobs:
       XSS_PROTECTION: True
       CSP_CONNECT_SRC: "*"
       CSP_FONT_SRC: "'self' https://fonts.gstatic.com https://fonts.googleapis.com https://code.cdn.mozilla.net https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/fonts/ data: https://static.fundraiseup.com/common-fonts/"
-      CSP_FRAME_SRC: "'self' https://www.google.com/recaptcha/ https://*.stripe.com https://pay.google.com https://*.paypal.com"
+      CSP_IMG_SRC: "* data: blob: *.fundraiseup.com ucarecdn.com pay.google.com *.paypalobjects.com"
```

Comment:
Let's be more specific and include the `https://` protocol for extra safety 😁 . Please also update other files as well.